.github/workflows/gem_deploy_ouroborus.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish Ouroborus Gem
+
+on:
+  push:
+    tags:
+      - 'ouroborus*'
+
+jobs:
+  publish-gem:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.0 
+
+      - name: Publish Ouroborus Gem to GitHub Registry
+        run: |
+          mkdir -p ~/.gem
+          echo ":github: Bearer ${{ secrets.GITHUB_TOKEN }}" > ~/.gem/credentials
+          chmod 0600 ~/.gem/credentials
+          cd ouroborus
+          gem build ouroborus.gemspec
+          gem push --key github --host https://rubygems.pkg.github.com/geoSales-evolution ouroborus-*.gem
```

Comment:
Dúvida básica, notei que colocou `geoSales` no host, esse parâmetro é case sensitive?

Reply:
I alsp used GeoSales-Evolution and it work. but when I pull the image it comes with `geosales-evolution`.